templates/repo/pulls/fork.tmpl

```diff
@@ -9,6 +9,13 @@
 				</h3>
 				<div class="ui attached segment">
 					{{template "base/alert" .}}
+
+					{{if not .CanCreateRepo}}
+						<div class="ui negative message">
+							<p>{{.locale.TrN .MaxCreationLimit "repo.form.reach_limit_of_creation_1" "repo.form.reach_limit_of_creation_n" .MaxCreationLimit}}</p>
```

Comment:
This logic looks somewhat incorrect:
You always assume that the root cause for not being able to create a repo is that the limit has been reached.
But I think for this generic variable, it can be false for a few more reasons.
Perhaps not now, but potentially in the future.